composer.json

```diff
@@ -15,21 +15,19 @@
         }
     ],
     "require": {
-        "sonata-project/admin-bundle": "2.3.* || ^3.0",
-        "stof/doctrine-extensions-bundle": "^1.2",
+        "friendsofsymfony/jsrouting-bundle": "^1.6",
         "knplabs/knp-menu-bundle": "^2.0",
-        "friendsofsymfony/jsrouting-bundle": "^1.6"
+        "sonata-project/admin-bundle": "^2.3 || ^3.0",
+        "stof/doctrine-extensions-bundle": "^1.2"
     },
-    "minimum-stability": "stable",
-    "prefer-stable": true,
```

Comment:
Could you please add the minimum stability and prefer-stable back.
Then I'll be able to accept this.